packages/widgets/src/dockpanel.ts

```diff
@@ -1400,7 +1400,7 @@ export namespace DockPanel {
      *
      * @returns A new tab bar for a dock panel.
      */
-    createTabBar(document?: Document | ShadowRoot): TabBar<Widget> {
+    createTabBar(document: Document | ShadowRoot): TabBar<Widget> {
```

Comment:
We should keep it optional as this is part of the public API.

```suggestion
    createTabBar(document?: Document | ShadowRoot): TabBar<Widget> {
```